R/colour-text.r

```diff
@@ -5,9 +5,13 @@ testthat_colours <- list(
   error = crayon::red
 )
 
-colourise <- function(text, as = c("passed", "skipped", "error")) {
+#' @export
+colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
```

Comment:
This will do it.

On Wed, Jan 7, 2015 at 4:41 PM, Robert Krzyzanowski <technoguyrob@gmail.com>
wrote:

> Just compare values. One second.
>
> On Wed, Jan 7, 2015 at 4:38 PM, Kirill Müller <notifications@github.com>
> wrote:
>
>> In R/colour-text.r
>> <https://github.com/hadley/testthat/pull/211#discussion-diff-22622471>:
>>
>> > @@ -5,9 +5,13 @@ testthat_colours <- list(
>> >    error = crayon::red
>> >  )
>> >
>> > -colourise <- function(text, as = c("passed", "skipped", "error")) {
>> > +#' @export
>> > +colourise <- function(text, as = c("passed", "skipped", "error"), fg, bg) {
>>
>> This doesn't work for a call like colourise("text", "green"). OTOH, we
>> really want to have as in the second position. @hadley
>> <https://github.com/hadley>: Any idea?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/hadley/testthat/pull/211/files#r22622471>.
>>
>
>
>
> --
> -Robert Krzyzanowski
>



-- 
-Robert Krzyzanowski